doc/nrf/app_dev/device_guides/nrf53/building_nrf53.rst

```diff
@@ -35,7 +35,7 @@ You can build and program separate images or combined images using the |nRFVSC|.
 
       Complete the following steps to build and program a multi-image build to the nRF5340 application core and network core:
 
-	   .. include:: ../../includes/vsc_build_and_run.txt
+	   .. include:: ../../../includes/vsc_build_and_run.txt
```

Comment:
```suggestion
      .. include:: /includes/vsc_build_and_run.txt
```
There's a tab at the start of the line, and since this is pointing to the `include` directory, it might as well be a hard link instead of relative.